owl/ease_home.owl

```diff
@@ -0,0 +1,447 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="http://www.ease-crc.org/ont/ease-home.owl#"
+     xml:base="http://www.ease-crc.org/ont/ease-home.owl"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+    <owl:Ontology rdf:about="http://www.ease-crc.org/ont/ease-home.owl"/>
```

Comment:
- import of SOMA.owl is missing
- a small description would be nice (annotation property)
- the name should be rather "SOMA-HOME"